internal/daysteps/daysteps.go

```diff
@@ -1,19 +1,63 @@
 package daysteps
 
 import (
-	...
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint5-final/internal/personaldata"
+	"github.com/Yandex-Practicum/go1fl-sprint5-final/internal/spentenergy"
 )
 
 const (
 	StepLength = 0.65
 )
 
-// создайте структуру DaySteps
-...
+var (
+	errLenSlice          = errors.New("slice length is not equal to 2")
+	errConvertToInt      = errors.New("invalid string to int conversion")
+	errConvertToDuration = errors.New("invalid string to duration conversion")
+	errDuration          = errors.New("duration len must be greater than zero")
+)
 
+// создайте структуру DaySteps
+type DaySteps struct {
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
+}
 
 // создайте метод Parse()
-...
+func (ds *DaySteps) Parse(datastring string) (err error) {
+	strSlice := strings.Split(datastring, ",")
+	if len(strSlice) != 2 {
+		return errLenSlice
+	}
+	steps, err := strconv.Atoi(strSlice[0])
+	if err != nil {
+		return errConvertToInt
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```